OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -972,10 +972,14 @@ private static PushRegistrator getPushRegistrator() {
       int deviceType = osUtils.getDeviceType();
       if (deviceType == UserState.DEVICE_TYPE_FIREOS)
          mPushRegistrator = new PushRegistratorADM();
-      else if (OSUtils.hasFCMLibrary())
-         mPushRegistrator = new PushRegistratorFCM();
+      else if (new OSUtils().getDeviceType() == UserState.DEVICE_TYPE_ANDROID) {
```

Comment:
You can use the new utils function

Reply:
oh I think the new util function is in the other PR 😢

Reply:
Added to this one, since it is a better fit anyway